bin/cli.js

```diff
@@ -91,7 +91,7 @@ function commandScript(cmd, opts) {
 
         return 'npx webpack --watch';
     } else if (cmd === 'watch' && opts.hot) {
-        return 'npx webpack serve --hot --inline --disable-host-check';
+        return 'npx webpack serve --hot';
```

Comment:
The `hot` option is left over because it checks if it is included in `process.argv`.
Let me know if there is a better way

Reply:
We should probably ask webpack itself if it's running via HMR. I'll take a look today to see how to do that.

Reply:
It will also be good if we can enable the webpack-dev-server https. I tried this `npx mix --hot --https` but got an` unknow option` error. it would be nice if we weren't limited to the options defined by laravel-mix

Reply:
We're not! You can pass arbitrary options after `--` like so `npx mix --hot -- --https`

Reply:
Thanks @thecrypticace. it worked but when I reload my page I get net::ERR_CERT_AUTHORITY_INVALID http error. Maybe I'm doing something wrong

Reply:
That's because webpack's https support uses a self-signed certificate by default. If you want to customize that you'll have to customize the webpack config.